src/lib.rs

```diff
@@ -242,27 +242,65 @@ pub struct Binding<'a> {
     pub resource: BindingResource<'a>,
 }
 
-/// Specific type of a binding..
+/// Specific type of a binding.
+/// WebGPU spec: https://gpuweb.github.io/gpuweb/#dictdef-gpubindgrouplayoutentry
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 pub enum BindingType {
+    /// A buffer for uniform values.
+    ///
+    /// Example GLSL syntax:
```

Comment:
this will need to be changed to WGSL soon :)

Reply:
True. But having some examples is better than none :p